src/pages/relicans.module.scss

```diff
@@ -0,0 +1,77 @@
+.section {
+    &:not(:last-child) {
+      margin-bottom: 4rem;
+    }
+  }
+  
+  .twoColumn {
+    display: grid;
+    grid-template-columns: repeat(2, calc(50% - 1rem));
+    grid-gap: 2rem;
+  
+    @media (max-width: 760px) {
+      grid-template-columns: 1fr;
+    }
+  }
+  
+  
+  .meetTheTeam {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 2rem;
+
+    @media (max-width: 1400px) {
+        grid-template-columns: repeat(2, 1fr)
+    }
+  
+    @media (max-width: 1200px) {
+        grid-template-columns: repeat(1, 1fr)
+    }
+  }
+
+  .socials {
+        list-style: none;
+  }
+
+  .socials > li {
+    display: inline;
+    margin: 0 5px;
+    padding: 0;
+  }
+
+  .socials > li:first {
+    margin-left: 0px;
+  }
+
+  .avatar {
+      max-width: 150px;
+      height: auto;
+      border-radius: 50%;
+  }
+  
+  .point {
+    text-align: center;
+    max-width: 400px;
+    min-height: 575px;
+    margin: auto;
```

Comment:
I would get rid of this `margin: auto;` because it's currently causing the tiles to be uneven. (Look at Jonan Scheffler's tile) I think you need to use something like `align-items` in order to center them on a mobile view.

Reply:
<img width="1056" alt="Screen Shot 2021-01-27 at 11 19 35 AM" src="https://user-images.githubusercontent.com/38332422/106020276-94ecbf80-6091-11eb-85b4-7a8bb75f2386.png">